### PR TITLE
Feature/process count control on windows

### PR DESCRIPTION
**Pull request by dothebart**
http://jenkins01.arangodb.biz:8080/job/arangodb-quicky-windows/EDITION=community,STORAGE_ENGINE=mmfiles,TEST_SUITE=single,limit=windows&&build/23/

### CODE REVIEW

File: helper.psm1 [hunk 1]
```diff
@@ -17,7 +17,22 @@ $env:TMP = "$INNERWORKDIR\tmp"
...
+$global:ok = $true
+
+################################################################################
+# Utilities
+################################################################################
+
+#ToDo
+#Function transformBundleSniplet
+#{   
+#}
```
Comment (fceller) [inline helper.psm1]: Why is this needed?

Comment (Luschgy) [reply helper.psm1]: It is just a reminder to implement this function

**Comment by dothebart** [comment]
http://jenkins01.arangodb.biz:8080/job/arangodb-quicky-windows/39/EDITION=community,STORAGE_ENGINE=mmfiles,TEST_SUITE=single,limit=windows&&build/console is the final run that has a good behaviour, so we should merge this now.

**Comment by Luschgy** [comment]
LGTM
